service/dps/storm/topologies/ic/ic-topology/src/main/resources/ic-topology-config.properties

```diff
@@ -15,8 +15,6 @@ STORM_ZOOKEEPER_ADDRESS=STORM_ZOOKEEPER_ADDRESS
 
 
 MCS_URL=MCS_URL
-MCS_USER_NAME=ic_user_name
-MCS_USER_PASS=ic_user_password
```

Comment:
It should be updated also on:

docker/dps/ic-topology-config.properties
docker/dps/xslt-topology-config.properties